A55-xds-stateful-session-affinity.md

```diff
@@ -421,14 +427,17 @@ or `ring_hash_experimental`) is created as the child policy of
 To propagate [`common_lb_config.override_host_status`][or-host-status] the
 following changes are required:
 
-* the [`XdsClient`][grpc-client-arch] to parse
+* CDS update message will be modified to include the `override_host_status`
+field.
+
+* the [`XdsClient`][grpc-client-arch] will parse
 [`common_lb_config.override_host_status`][or-host-status] and copy the value
 in its CDS update to the watchers.
 
-* `cds_experimental` policy (which receives the CDS update) copies the value
-of `override_host_status` into the config of `xds_override_host_experimental`
-and embed this config in the `xds_lb_policy` field of the config of
-`xds_cluster_resolver_experimental`.
+* `cds_experimental` policy (which receives the CDS update) will copy the
```

Comment:
We need one more bullet after this one:

"""
When the `xds_cluster_resolver_experimental` policy generates the config for each priority, it will configure the `xds_override_host_experimental` policy underneath the `xds_cluster_impl_experimental` policy.  The generated config for the `xds_override_host_experimental` policy will include the `override_host_status` field from the `DiscoveryMechanism` associated with that priority.
"""

Reply:
done